app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
```

Comment:
- [ ] Upon registering for the user, I'm unable to to successfully  register and login. I have looked at the terminal to see if there is a confirmation link for verification of the account but I did not find any. Kindly look at the screenshot below to appreciate the bug. 

### Note: User is not able to be added to the database

![Screenshot from 2022-08-31 11-54-07](https://user-images.githubusercontent.com/46388113/187674142-90b12a6d-995c-434d-b4e5-0e2945dbd1cd.png)